openbas-model/src/main/java/io/openbas/database/repository/ExerciseRepository.java

```diff
@@ -103,17 +127,17 @@ List<RawInjectExpectation> allGrantedInjectExpectationsFromDate(@Param("from") I
           +
           "FROM exercises " +
           "INNER JOIN injects ON exercises.exercise_id = injects.inject_exercise " +
-          "JOIN injects_expectations ie ON injects.inject_id = ie.inject_id " +
+          "LEFT JOIN injects_expectations ie ON injects.inject_id = ie.inject_id " +
```

Comment:
I needed a LEFT JOIN oterhwis in some case I do not have any data in the mitre aray in dashboard